docker-compose.yaml

```diff
@@ -22,12 +34,12 @@ services:
   generator:
     image: killrvideo/killrvideo-generator:1.2.4
     depends_on:
-    - dse
-    - etcd
+      - dse
+      - etcd
     environment:
       KILLRVIDEO_ETCD: "etcd:2379"
-      KILLRVIDEO_DSE_USERNAME: $KILLRVIDEO_DSE_USERNAME
-      KILLRVIDEO_DSE_PASSWORD: $KILLRVIDEO_DSE_PASSWORD
+      KILLRVIDEO_DSE_USERNAME: cassandra
```

Comment:
KILLRVIDEO_DSE_USERNAME and KILLRVIDEO_DSE_PASSWORD values are hardcoded to "cassandra" in docker-compose.yaml file. These values are dynamic and are controlled via environment variables. There are multiple use-cases, especially when dealing with external clusters, where we cannot use the default username and password.